Italian/scappare/scopri-oggetto/esamina-stanza/esamina-fogli/esamina-fogli.md

```diff
@@ -0,0 +1,2 @@
+I fogli sono tutti bianchi.
+Mentre osservi la scrivania noti un cassetto. Curiosamente, al posto della maniglia è presente solo un intaglio di forma triangolare.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.